.gitignore

```
@@ -0,0 +1,3 @@
+/.idea
+/cmake-build-debug
+/build
```

```suggestion
.idea
cmake-build-debug
build
```

I would also move this change to a separate commit, with a message like "Add .gitignore".

---

I removed .gitignore from this pull request, and will prepare another :)